src/displayapp/DisplayApp.cpp

```diff
@@ -441,45 +441,45 @@ void DisplayApp::LoadApp(Apps app, DisplayApp::FullRefreshDirections direction)
     case Apps::StopWatch:
       currentScreen = std::make_unique<Screens::StopWatch>(this, *systemTask);
       break;
-#if APP_TWOS
     case Apps::Twos:
-      currentScreen = std::make_unique<Screens::Twos>(this);
+      if (APP_TWOS) {
```

Comment:
If I understand `if conxtexpr()` correctly, it is evaluated at compile time.
So I think it would be a better idea to use it here:
```cpp
if constexpr(APP_TWOS) {
// ...
```

Reply:
That is indeed a good idea, however that feature has only been added in C++17, and this project is currently set to C++14. I think I remember some talk about switching to C++20 at some point though.

Reply:
I just commited another implementation that needs C++17. Maybe it's time :wink: Thanks for the hint, I will have a look at it!